.github/workflows/maven-build-caching-samples-verification.yml

```diff
@@ -31,7 +31,7 @@ jobs:
             -c "$GITHUB_SHA" \
             -p "build-caching-maven-samples" \
             -g "verify" \
-            -a "-Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug -B" \
+            -a "-Dorg.slf4j.simpleLogger.log.gradle.goal.cache=debug -Dorg.slf4j.simpleLogger.log.gradle.goal.fingerprint=trace -B" \
```

Comment:
oh, what does this flag do and how does it help here?

Reply:
This is just for now to find the issue with the action that fails and the validation scripts. Will revert it later once all is fixed.

Reply:
Reverted in https://github.com/gradle/gradle-enterprise-build-config-samples/pull/546/commits/64b57eb538c932ee0b3c9810cb58abd7ef00b343